salt/metalk8s/orchestrate/control_plane.sls

```diff
@@ -0,0 +1,48 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+{% set node = pillar.get('node_name', 'node1') %}
+
+Update mine:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.mine.ips
+
+Bootstrap client certs:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+          - metalk8s.kubeadm.init.certs.apiserver
+          - metalk8s.kubeadm.init.certs.apiserver-kubelet-client
+          - metalk8s.kubeadm.init.certs.sa
```

Comment:
Not sure this'll work. I believe the ServiceAccount key-pair needs to be *shared* across all `k-c-m` nodes, it's not really a CA-based system. To be validated, though.

Reply:
Fixed. Now we deploy the SA CA cert everywhere?